packages/mendel-outlet-browser-pack/src/index.js

```diff
@@ -65,10 +72,12 @@ module.exports = class BrowserPackOutlet {
             let prelude = '';
             let appendix = '';
 
-            if (hasProcess) {
-                prelude = '(function(){var process={env: {}};';
+            if (hasProcess || hasGlobal) {
+                prelude = '(function(){';
                 appendix = '})();';
             }
+            if (hasProcess) prelude += 'var process={env: {}};';
+            if (hasGlobal) prelude += 'var global=window;';
```

Comment:
shouldn't this be something like:

```js
if (hasGlobal || hasProcess) prelude += 'var global=window;';
if (hasProcess) prelude += 'var process=global.process||{env: {}};';
```
This would be a better polyfill for process because process is the same instance across all modules.